notification/webhook/sender.go

```diff
@@ -57,6 +57,16 @@ type POSTDataVerification struct {
 	Code    string
 }
 
+// POSTDataOnCallNotification represents fields in outgoing on call notification.
+type POSTDataOnCallNotification struct {
+	AppName      string
+	Type         string
+	Users        []notification.User
```

Comment:
Because this struct defines an API, let's add a struct def here instead of re-using one. Otherwise, changing a different package (`notification.User`) could result in a difference in our API without a compiler error.

Adding an explicit def here also has the benefit of having all webhook API code and fields in one place.

```go
// We use types defined in this package to insulate against unintended API
// changes.
users := make([]POSTDataOnCallUser, len(m.Users))
for i, u := range m.Users {
	users[i] = POSTDataOnCallUser(u)
}
```

Since the structs are identical, we can do a direct assignment, but this will insulate from tag changes in the notification package and create a compiler error if there are any incompatible changes.

Reply:
That makes sense, thanks for the explanation. I just made the change